### DISCUSSION TITLE
Update to Weewx 4.10.1 and weewx-wdc stopped working

### DISCUSSION THREAD

**Discussion by JonFear** [Daveiano/weewx-wdc#110] [General]
Hi!

Your favorite pain in the A**.

Applied the update from Weewx 4.9.1 to 4.10.1 and the skin has stopped working. I have Belchertown running in parallel and it is still working.

Any thoughts? Is there any log files I can copy for review?

Thanks

Jon

**Comment by Daveiano** [comment]
Sorry for the struggling with the update!

For log files please have a look at https://github.com/weewx/weewx/wiki/Help!-Posting-to-weewx-user#the-system-log 
Would be great to get more details!

**Comment by JonFear** [comment]
So far not much but...

jon@weewx-cctv:~ $ sudo tail -f /var/log/syslog | grep wdc
Feb  1 20:50:18 weewx-cctv weewx[81061] DEBUG weewx.reportengine: Found configuration file /etc/weewx/skins/weewx-wd 
/skin.conf for report 'WdcReport'
Feb  1 20:50:18 weewx-cctv weewx[81061] DEBUG weewx.cheetahgenerator: Using search list ['user.weewx_wdc.WdcGeneralUtil', 'user.weewx_wdc.WdcStatsUtil', 'user.weewx_wdc.WdcDiagramUtil', 'user.weewx_wdc.WdcCelestialUtil', 'user.weewx_wdc.WdcArchiveUtil', 'user.weewx_wdc.WdcTableUtil', 'weewx.cheetahgenerator.Almanac', 'weewx.cheetahgenerator.Current', 'weewx.cheetahgenerator.DisplayOptions', 'weewx.cheetahgenerator.Extras', 'weewx.cheetahgenerator.Gettext', 'weewx.cheetahgenerator.JSONHelpers', 'weewx.cheetahgenerator.PlotInfo', 'weewx.cheetahgenerator.SkinInfo', 'weewx.cheetahgenerator.Station', 'weewx.cheetahgenerator.Stats', 'weewx.cheetahgenerator.UnitInfo']
Feb  1 20:50:18 weewx-cctv weewx[81061] ERROR weewx.reportengine:         ****    File "/usr/share/weewx/user/weewx_wdc.py", line 25, in <module>

**Comment by Daveiano** [comment]
Thank you for your help, I was able to find the bug. This should be resolved in the latest [release](https://github.com/Daveiano/weewx-wdc/releases/tag/v2.3.3) 

You can just comment here if you encounter more problems.

**Reply by JonFear** [reply]
Perfect!

Thank you...

Jon

**Comment by JonFear** [comment]
Fixed and new issue supplied, thank you. House keeping task, closing subject.
